middleware/body_limit.go

```diff
@@ -102,9 +101,8 @@ func (r *limitedReader) Close() error {
 	return r.reader.Close()
 }
 
-func (r *limitedReader) Reset(reader io.ReadCloser, context echo.Context) {
```

Comment:
we can not change public API like that - this would be backward incompatible change

Reply:
nevermind - this is not public struct